Configuration/Eras/python/Modifier_PF_badHcalMitigation_cff.py

```diff
@@ -0,0 +1,3 @@
+import FWCore.ParameterSet.Config as cms
+
+PF_badHcalMitigation =  cms.Modifier()
```

Comment:
please start with a lower case as for all other modifiers.
Sorry for mentioning this late.
I have a memory of making this comment a week ago, but have no evidence that it was actually posted.

Reply:
changed `PF_badHcalMitigation` -> `pf_badHcalMitigation`